docs/usage/cli/distance.rst

```diff
@@ -54,6 +54,8 @@ The `map` key specifies a dictionary of weights to use for distance calculations
 These weights are indexed hierarchically by gene name and one-based gene coordinate and are assigned in either a sequence-independent or sequence-dependent manner.
 The simplest possible distance map calculates Hamming distance between sequences without any site-specific weights, as shown below::
 
+.. code-block:: json
+
```

Comment:
The [following commit](https://github.com/nextstrain/augur/pull/1095/commits/1b4de339a7efb47c0026ec15bd03f00ca2242174) should be part of this change, as the sentence-ending `::` is [syntax for introducing the next paragraph as literal block](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#rst-literal-blocks) (e.g. similar to `.. code-block::` with no language hint).

Reply:
Fixed in 2a47e09f12e0f316287da4485281838e383ba323.